.gitignore

```diff
@@ -59,6 +59,4 @@ dist-ssr
 .aider*
 .coverage
 
-backend/README.md
-
-myenv
+backend/README.md
```

Comment:
Trailing newline issue. https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Sorry for dropping the ball on this a bit, but sometimes too much is happening to do community work :( 
All the work has been done on my old private laptop, so yeah, my editor was just set up quick and dirty.. Sorry about that.

Im more than happy to hand this over to someone with more time, as im quite busy two more weeks